.travis.yml

```diff
@@ -1,19 +1,31 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
When still running <= 5.4 you won't be able to install this version. So no breaking application. Zend Framework also dropped 5.3 and 5.4 within their minor releases.